bundles/org.openhab.binding.tr064/src/main/resources/channels.xml

```diff
@@ -138,74 +138,26 @@
 		<getAction name="GetInfo" argument="NewEnable"/>
 		<setAction name="SetEnable" argument="NewEnable"/>
 	</channel>
-	<channel name="macOnline" label="MAC Online" description="Online status of the device with the given MAC">
+	<channel name="macOnline" label="MAC Online" description="Online status of the device with the given MAC"
+		advanced="true">
 		<item type="Switch"/>
 		<service deviceType="urn:dslforum-org:device:LANDevice:1" serviceId="urn:LanDeviceHosts-com:serviceId:Hosts1"/>
 		<getAction name="GetSpecificHostEntry" argument="NewActive">
 			<parameter name="NewMACAddress" thingParameter="macOnline"
 				pattern="([0-9A-Fa-f]{2}:){5}[0-9A-Fa-f]{2}(\s*#.*)*"/>
 		</getAction>
 	</channel>
-	<channel name="macIP" label="MAC IP" description="IP of the device with the given MAC">
+	<channel name="macOnlineIpAddress" label="MAC Online IP" description="IP of the device with the given MAC"
```

Comment:
Is this a channel rename? The README has changed the description to "IP of the MAC (uses same parameter as `macOnline`)"

Reply:
Yes. This channel was added to the other version before it was added here. That's why I renamed it.

Reply:
OK, please remember to mention this in https://github.com/openhab/openhab-distro/blob/main/distributions/openhab/src/main/resources/bin/update.lst.

Should the description here be updated to be in line with the description in the README? In this PR this has changed from "IP of the device with the given MAC" to "IP of the MAC (uses same parameter as `macOnline`)", probably for a reason?

Reply:
Yes, Thera have been questions how to configure that channel, that's why I added a clarification.

I wonder if we should also rename the other channels that act on the same list of parameters (`macSignalStrength1` and similar) for consistency.